packages/ui-kit/src/design-system.ts

```diff
@@ -288,8 +288,9 @@ export const dimensions = {
 
 export const typography = {
   fontFamilies: {
-    primary: "'Roboto', sans-serif",
+    primary: "'Inter var', sans-serif",
     code: "'Roboto Mono', monospace",
+    nonMonospace: "'Inter var'",
```

Comment:
to be honest I find the name of the token not ideal.  It's not actually a statement about what it is, it's just excluding monospace but leaving anything else undescribed

Reply:
Any suggestions?